Overview.src.html

```diff
@@ -3789,18 +3789,17 @@ <h4 id="target-properties">Target properties</h4>
 Each <a>keyframe effect</a> can have zero or more associated
 <dfn lt="target property">target properties</dfn>.
 
-<a lt="target property">Target properties</a> may be CSS properties
+<a lt="target property">Target properties</a> may refer to CSS properties
 or DOM attributes.
-If a given <a>target element</a> has an attribute with the same name
-as a CSS property, any <a>target property</a> of that name is taken to
-refer to to the CSS property.
 
-Note: If there ever exists a situation where we need to animate an attribute
-      with the same name as a property (other than a <a
-      href="http://www.w3.org/TR/SVG2/intro.html#TermPresentationAttribute">presentation
-      attribute</a> [[SVG2]]) then we will need to introduce
-      a disambiguation strategy. Generally, however, such naming should be
-      avoided.
+If a target property refers to a DOM attribute, that attribute must be an SVG
+DOM attribute. The target property's name will consist of the attribute name, prefixed
+with "svg", and with the attribute's first letter capitalized.
```

Comment:
Also, the first sentence about "that attribute must be an SVG DOM attribute" seems awkward somehow. Perhaps, "User agents are only required to support animation of attributes on SVG elements"? Perhaps with a further explanation that this only applies to user agents that support SVG at all.